thoth/common/logging.py

```diff
@@ -149,6 +149,29 @@ def _get_sentry_integrations() -> List[object]:
     return integrations
 
 
+def before_send_handler(event, hint):
+    """Filter the errors caught before sending to Sentry.
+
+    This function ignores the exceptions passed in as a environment variable in a comma separated manner.
+    """
+    ignored_exceptions = os.getenv('THOTH_SENTRY_IGNORE_EXCEPTION')
+    if ignored_exceptions:
+        exceptions_split = ignored_exceptions.split(',')
+        if 'exc_info' in hint:
+            exc_type, exc_value, tb = hint['exc_info']
+            for exception in exceptions_split:
+                if exception == exc_type.__name__:
+                    _LOGGER.info(hint)
```

Comment:
No `_LOGGER.info` should be needed here - the exception will be printed to standard error as expected but it will not be sent to sentry.

Reply:
Okay will remove it.